Data-Analytics/Spark/DataProcessTransfer/DataProcessTransferFts.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: "sparkoperator.hpe.com/v1beta2"
+kind: SparkApplication
+metadata:
+  name: data-process-transfer-demo
+  namespace: <username>
+spec:
+  sparkConf:
+    spark.hadoop.fs.s3a.path.style.access: "true"
+    spark.hadoop.fs.s3a.impl: "org.apache.hadoop.fs.s3a.S3AFileSystem"
+    spark.hadoop.fs.s3a.connection.ssl.enabled: "false"
+    spark.mapr.extraconf.secret: "<k8s-secret-name>"
+    spark.hadoop.fs.s3a.bucket.ezua-demo.endpoint: http://local-s3-service.ezdata-system.svc.cluster.local:30000
+    spark.driver.extraJavaOptions: "-Dcom.amazonaws.sdk.disableCertChecking=true"
+    spark.executor.extraJavaOptions: "-Dcom.amazonaws.sdk.disableCertChecking=true"
+  type: Scala
+  sparkVersion: 3.5.0
+  mode: cluster
+  image: gcr.io/mapr-252711/spark-3.5.0:v3.5.0
+  imagePullPolicy: Always
+  mainApplicationFile: "local:///ezua-tutorials/Data-Analytics/Spark/DataProcessTransfer/DataProcessTransfer.jar"
```

I see you added `/mounts/shared-volume` mount. So here it should be:
```suggestion
  mainApplicationFile: "local:///mounts/shared-volume/ezua-tutorials/Data-Analytics/Spark/DataProcessTransfer/DataProcessTransfer.jar"
```